hw/top_earlgrey/cdc/cdc_waivers.tcl

```diff
@@ -92,3 +92,6 @@ set_rule_status -rule {W_CNTL} -status {Waived} \
 set_rule_status -rule {W_DATA} -status {Waived} \
   -expression {(Signal=~"*u_pinmux_aon.dio_pad_attr_*")} \
   -comment {PAD Attributes are static signals.}
+
+# ndmreset
+set_rule_status -rule {W_CNTL} -status {Waived} -expression {(ReceivingFlop=~"*.u_rstmgr_aon.u_ndm_sync*") && ((Signal=~"*.u_rv_dm.*ndmreset") || (Signal=~"*u_rv_dm.u_lc_en_sync*"))} -comment {LC Sync remains the value until escalation ndmreset only affects the rstmgr}
```

Comment:
@eunchan should i actually fix this in the design?
the path right now is flop -> AND gate -> synchronizer.

I know technically it doesn't matter because it's escalate, but i think it would be cleaner to fix.
What do you think?

Reply:
yep. Would be good if we move it prior to the flop.

Reply:
sounds good let me make the fix.